zeeguu/api/api/nlp.py

```diff
@@ -58,3 +59,28 @@ def annotate_clues():
 
     return json_result(updated_words)
 
+# ---------------------------------------------------------------------------
+@api.route("/get_sentences_for_wo", methods=("POST",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def get_sentences_for_wo():
```

Comment:
Currently, the way I implemented it is within the OrderWords exercise itself - so if the context is too long, then the articleInfo is used to find a new sentence that is smaller. So at that point, we only get the article associated with that bookmark. However, the issue with this is the fact that they might not see the word again, so we would have to change the outcome to not update the tries with the word associated. 

I am hoping the method that just selects a subsentence works well because then it would avoid this type of cases. 

In sum:
Inputs: Article Text, Bookmark Context, Language
Outputs: Heap with the Similarities based on the spaCy embeddings [(similarity, Sentence), ...]